lldb/bindings/python/python-wrapper.swig

```diff
@@ -621,6 +623,9 @@ bool lldb_private::python::SWIGBridge::LLDBSwigPythonCallCommand(
     pfunc(debugger_arg, PythonString(args),
           SWIGBridge::ToSWIGWrapper(std::move(exe_ctx_ref_sp)), cmd_retobj_arg.obj(), dict);
 
+  if (PyErr_Occurred())
```

Comment:
Might be nice here to check if the CommandReturnObject's ErrorString is empty, and if it is, add a boiler-plate error message like "Scripted command threw an uncaught exception".

Reply:
Since warnings are also written to the error stream, should different logic be used? For example, set an error message if the status is not Failed?

Reply:
Maybe the answer is as simple as calling `AppendError`, that way we get both messages.

Reply:
Yes, appending a command-centered message to the error stream sounds right to me.  

A raw dump of a python expression - especially if you don't really know that lldb has python implemented commands - will just be confusing.  But if we have the exception dump followed by a line attributing it to this lldb command, that would clearer.

It doesn't look like you know the actual command name at this point, but even saying the python function or class name might help people figure it out.

Reply:
I changed the message to include function or class name.